apps/homepage/src/app/blog/components/route/home/home.component.ts

```diff
@@ -0,0 +1,91 @@
+import { HttpClient } from '@angular/common/http';
+import { Message } from '@angular/compiler/src/i18n/i18n_ast';
+import { Component, OnInit } from '@angular/core';
+import { TransferStateService } from '@scullyio/ng-lib';
+import { Apollo, gql } from 'apollo-angular';
+import { map } from 'rxjs/operators';
+
+export interface Article {
+  title: string;
+  body: string;
+  publishing: Date;
+  writer: Writer;
+  tags: Tag[];
+}
+
+export interface Writer {
+  name: string;
+  jobTitle: string;
+  articles: Article[];
+}
+
+export interface Tag {
+  name: string;
+  articles: Article[];
+}
+
+@Component({
+  selector: 'web-page-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  constructor(
+    private apollo: Apollo,
+    private transferState: TransferStateService
+  ) {}
+
+  private readonly articleQueryRef$ = this.apollo.watchQuery<{
+    articles: Article[];
+  }>({
+    query: gql`
+      query ArticlesProviding($where: ArticleWhereInput) {
+        articles(orderBy: createdAt_DESC, where: $where) {
+          title
+          publishing
+          writer {
+            name
+          }
+          tags {
+            name
+          }
+        }
+      }
+    `,
```

Comment:
It's a good idea to reuse it in a single query.
```suggestion
  private readonly articleQueryRef$ = this.apollo.watchQuery<{
    articles: Article[];
    tags: Tag[];
  }>({
    query: gql`
      query ArticlesProviding($where: ArticleWhereInput) {
        articles(orderBy: createdAt_DESC, where: $where) {
          title
          publishing
          writer {
            name
          }
          tags {
            name
          }
        }
      }
      tags {
        name
      }
    `,
```

Reply:
good idea.
Let's implement.